config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/BetterQuesting-AAAAAAAAAAAAAAAAAAAF3A==.json

```diff
@@ -31,7 +31,7 @@
+      "desc:8": "Better Questing has a number of useful commands for the questbook (QB). You need to have cheats enabled to use them. \n\n/bq_admin default load§r\nUse this command to update your QB after you update the modpack. It reloads the quest database from the file. Your save file stores a version of the QB that can be different from the one in the file.\n\n§//bq_admin edit\nToggles editing quests. Allows you to see all quests and their requirements§r on hover, like the View Mode§r button. RMB on the quest will open a context menu to share it or copy its ID.\n\n/bq_admin reset all \u003cPlayer\u003e§r\nResets all quests. Use this if you want to start over, or are editing the QB. You should specify the player as well if it is a server, otherwise you can skip that part.\n\n/bq_admin reset \u003cID\u003e \u003cPlayer\u003e§r\nResets a specific quest, defined by the quest ID. Sometimes it may be easier to go into Edit Mode \u003e Designer Tool to specify what to reset.\n\n/bq_admin complete \u003cnumber\u003e \u003cPlayer\u003e§r\nCompletes a specific quest, similarly to the above. You can still claim the rewards. Use this command if a quest is broken, and then report it on GitHub§r, but please check if it\u0027s already reported.\n\n/bq_admin default save§r\nSaves changes to the QB after editing it. Make sure to turn edit mode off first and reset all quests, because it saves both of them.\n\n/bq_copyquests \u003cPlayerWithQuestsCompleted\u003e \u003cNewPlayer\u003e§r\nIf you want to sync the quests with someone in your party, then use this. They won\u0027t get any rewards for that though.\n\n/bq_admin hardcore§r\nEnables BQ3 Hardcore Mode. You only start with 3 lives, so best of luck. Additional life hearts are available in the Kill all the Things tab.\n\nAside from the consume-type quests, a quest counts for everyone in the party if one person finished it. However, if someone joined your party later, you need to get them up to speed first either through /bq_copyquests above, or manually. Otherwise, your quest completion won\u0027t count for them because the requirements for that quest are not fulfilled for them.\n\nIf your singleplayer world does not have cheats enabled, go to the ESC screen, open to LAN, then enable cheats. This makes pausing impossible until you reload afterwards. If you host a server, you can run the commands in the command line, without \"/\". You can also edit the level.dat file, and set \u0027allowCommands\u0027 to 1, but make sure to backup your world before doing that.\n\n§4For quests to save properly, you must be online and logged into your Mojang account when starting the game. Sorry, it\u0027s a limitation of Better Questing.§r"
```

Comment:
edit: I wouldnt recommend edit mode to players to 'view all quests and their requirements'. That is not what its for and will just lead to them not being able to complete quests anymore. just say its for editting quests instead.

save: does not save edit mode or progress. just remove that comment.

Reply:
ok, got it, working on it